docs/essentials/actions.md

```diff
@@ -17,25 +17,23 @@ The actions addon is used to display data received by event handler (callback) a
 
 Actions work via supplying special Storybook-generated “action” arguments (referred to as "args" for short) to your stories. There are two ways to get an action arg:
 
-### Action argType annotation
+### Via @storybook/test fn spy function
 
-You can use [argTypes](../api/argtypes.md) to tell Storybook that an arg to your story should be an action. Usually, it makes sense to do this at the component level (although you can apply it per individual story):
+The first and recommended way to write actions, is to use the `fn` function from `@storybook/test`. This package provides a utility to help mock and spy args, which is very useful for writing tests with the [play function](../writing-stories/play-function.md). You can mock your component's methods by assigning them with the `fn()` function:
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'angular/button-story-onclick-action-argtype.ts.mdx',
-    'web-components/button-story-onclick-action-argtype.js.mdx',
-    'web-components/button-story-onclick-action-argtype.ts.mdx',
-    'common/button-story-onclick-action-argtype.js.mdx',
-    'common/button-story-onclick-action-argtype.ts.mdx',
+    'angular/button-story-onclick-action-spy.ts.mdx',
+    'common/button-story-onclick-action-spy.js.mdx',
+    'common/button-story-onclick-action-spy.ts.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-When Storybook sees this argType, it will create an arg set to a special “action” callback. If your component calls this arg (based on the user's interaction or through the `play` function), the event will show up in the action panel:
+If your component calls an arg which is spied on (based on the user's interaction or through the `play` function), the event will show up in the action panel:
```

Comment:
```suggestion
If your component calls an arg (because of either the user's interaction or the `play` function) and that arg is spied on , the event will show up in the action panel:
```